demosplan/DemosPlanUserBundle/Logic/PublicAffairsAgentHandler.php

```diff
@@ -69,7 +69,7 @@ public function getFromResourceIdentifierObject(ResourceIdentifierObject $resour
         $expectedType = 'publicAffairsAgent';
         $actualType = $resourceIdentifierObject->getType();
         if ($expectedType !== $actualType) {
-            throw new InvalidArgumentException("expected ${expectedType} for all resourceIdentifierObjects, got ${actualType}");
```

Comment:
I've seen this now a couple of times and I don't get it. This looks to me like it should have never worked.